.gitignore

```diff
@@ -0,0 +1,2 @@
+# Main project configuration file.
+/config
```

Comment:
We should probably stick to the Unix convention of having all lines end in a linefeed: https://unix.stackexchange.com/a/18789